File: lei_lds2122_grupo2-main/Backend/Data/ViewModels/Response/FollowVM.cs
using Backend.Data.ViewModels.Types;

namespace Backend.Data.ViewModels
{
    /// <summary>
    /// View Model class for followers and following reqeust
    /// </summary>
    public class FollowVM
    {
        public List<FollowType> Follow{get; set;}
        
        /// <summary>
        /// Class' constructor method
        /// </summary>
        /// <param name="follow">List of users (followers or following)</param>
        public FollowVM(List<FollowType> follow){
            this.Follow = follow;
        }
    }
}

File: Frontend/src/app/viewModels/responses/FollowVM.ts
import { FollowType } from "../Types/FollowType";

/**
 * View Model that represents a list of the following users.
 */
export class FollowVM {

    /**
     * Construct's an instance of {@link FollowVM}
     * 
     * @param follow List of following users
     */
    constructor(
        public follow: Array<FollowType>
    ) { }
}